community/sharing.md

```diff
@@ -0,0 +1,51 @@
+# Sharing your Mods
```

Comment:
Missing recommended forum post content – description, screenshot etc

Reply:
Yeah, I was going to add screenshot yesterday but forgot. What GIF program do you recommend for Linux?

Reply:
I have ffmpeg, but that’s probably not what most people are gonna use. I have no idea, I only use text-interface tools for a lot of things.

Reply:
Well does it export GIF?

Reply:
It can handle a lot of file types, including gif.

Reply:
Ok, I will mention it then.